Client/Frontend/Browser/BrowserActivityViewController.swift

```diff
@@ -0,0 +1,39 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import UIKit
+
+class BrowserActivityViewController: UIActivityViewController {
```

Comment:
I put this into its own class because the `BrowserViewController` class is already quite large.